teaser/project.py

```diff
@@ -1059,6 +1060,11 @@ def export_aixlib(
         report: boolean
             if True a model report in form of a html and csv file will be
             created for the exported project.
+        export_vars: dict[str:list]
+            dict where key is a name for this variable selection and value is a
+             list of variables to export, wildcards can be used, multiple
+             variable selections are possible.
```

Comment:
Why the whitespace in the second and third line?

Reply:
Link to claytex o.s.? 
Also, explicitly note that this will only work for Dymola?

Reply:
you are right! both solved with 7cd15517